### DISCUSSION TITLE
Unable to connect to Isar inspector

### DISCUSSION THREAD

**Discussion by TNorbury** [isar/isar#357] [Help]
Hello,

I'm trying to connect Isar Inspector to my Isar instance. I open Isar with `inspector` set to true and copy the full url into Isar Inspector, press connect, then nothing. I'm able to connect to the VM if I enter the url (minus `ws://`) into my browser. I'm building my app for Mac, not sure if that impacts anything (or if there is some project configuration I need to do)

Specs:
MacOS
Flutter 2.10.5
Isar 2.4.0

**Comment by simc** [comment]
Let me check 🙂

**Comment by Mihai-B** [comment]
I thought this might be an issue on my machine, but I experience the same problems as @TNorbury  but on a Linux machine.
My Inspector terminal logs this error
```
flutter: ERROR: Unable to connect to VMService
flutter: ext.isar.listInstances: (-32000) Server error
Converting object to an encodable object failed: _LinkedHashSet len:1
flutter: #0      new _OutstandingRequest (package:vm_service/src/vm_service.dart:1718:45)
#1      VmService._call (package:vm_service/src/vm_service.dart:2225:21)
#2      VmService.callServiceExtension (package:vm_service/src/vm_service.dart:2190:14)
#3      Service._call (package:isar_inspector/service.dart:36:35)
#4      Service.listInstances (package:isar_inspector/service.dart:51:29)
#5      _ConnectPageState._connect (package:isar_inspector/connect.dart:25:39)
<asynchronous suspension>
```
I cann however connect to the VM machine if I input the url in browser without ws part.

Specs
```
Flutter is already up to date on channel stable
Flutter 2.10.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 5464c5bac7 (10 days ago) • 2022-04-18 09:55:37 -0700
Engine • revision 57d3bac3dd
Tools • Dart 2.16.2 • DevTools 2.9.2
Isar 2.4.0
```

**Reply by TNorbury** [reply]
I had been running the compiled version of Inspector, I'll try building it myself and see what sorta error message I get.

**Reply by TNorbury** [reply]
Got the same error

**Comment by simc** [comment]
This is fixed in the latest release :+1:
